src/pages/faq.module.scss

```diff
@@ -9,6 +9,10 @@
   }
 }
 
+.faqSection:first-of-type {
```

Comment:
oh wow TIL

Reply:
Yeah, it popped up in VSCode and proved easier to implement than programmatically adding a class when index = or != 0 😆